app/contributing/index.md

```diff
@@ -4,53 +4,42 @@ no_version: true
 layout: 'docs-v2'
 ---
 
-Hello, and welcome! Thanks for thinking about contributing to the Kong documentation.
-
-This section of the docs is here to help you help us, so read on to learn how to ask questions and help effectively.
-We respect your time, and the more you follow our guidelines the easier it is for us (docs maintainers) to respond
-promptly and help you get your pull requests merged.
+Hello, and welcome! Thanks for your interest in contributing to the Kong docs. The following information will help you contribute effectively, which will in turn help us to get your pull requests merged faster. 
```

Comment:
I actually think that this rephrase takes away something important from the intro. The original intro, from Jennifer, is really trying to get across the idea that we're not arbitrarily pushing rules onto the contributors, and have everyone's best interests at heart.